### ISSUE TITLE
[Enhancement] Make a different tab for Privacy FAQs

### ISSUE THREAD

**Issue by chandrikadeb7** [nic-delhi/AarogyaSetu_Android#261]
In the current version, the privacy FAQs are found by scrolling the status homepage tab. A separate tab/option can be made for FAQs.

**Comment by legendavey** [comment]
Makes sense. If this is implemented, it could also make room for opting-out of certain features in the future.
